Lib/test/test_fstring.py

```diff
@@ -628,23 +628,34 @@ def test_mismatched_parens(self):
                             r"does not match opening parenthesis '\('",
                             ["f'{a(4}'",
                             ])
-        self.assertRaises(SyntaxError, eval, "f'{" + "("*500 + "}'")
+        self.assertRaises(SyntaxError, eval, "f'{" + "("*20 + "}'")
```

Comment:
This looks worrying. Did the limit for nested parentheses go down over an order of magnitude?